binding.cc

```diff
@@ -1015,6 +1025,86 @@ namespace zmq {
 
 #endif
 
+  NAN_METHOD(Socket::Readv) {
+    Socket* socket = GetSocket(info);
+    if (socket->state_ != STATE_READY)
+      return info.GetReturnValue().Set(Nan::Undefined());
+
+    int events;
+    size_t events_size = sizeof(events);
```

Comment:
That seems debatable. Everywhere else in this file there is no space used, and I don't see it [here](http://en.cppreference.com/w/cpp/language/sizeof) either for example. Now I'm not one to care too much, but let's just say time-and-place ;) I don't mind putting this on the style overhaul checklist.